addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -149,7 +149,7 @@ export class ColorPlugin extends Plugin {
                 const hasAnySelectedNodeColor = (mode) => {
                     const nodes = this.dependencies.selection
                         .getTraversedNodes()
-                        .filter(isTextNode);
+                        .filter((n) => isTextNode(n) || n.classList.contains("o_selected_td"));
```

Comment:
The thing is the color plugin should not be aware of the `o_selected_td` thing which is exclusive to table. Bit I had a look and that is an issue that already exists in format so I guess the cat's out of the bag. I'm r+ing this PR and I'll create another task to change that.